src/Ldap.php

```diff
@@ -972,6 +982,8 @@ public function bind($username = null, $password = null, $saslOpts = null)
             if ($bind !== false) {
                 $this->boundUser = $username;
                 return $this;
+            } else {
+                $this->boundUser = false;
             }
```

Comment:
```suggestion
            }
            $this->boundUser = false;
```

As the `if` condition will **always** return we can skip the `else` here.

Reply:
Will do. Personally I feel that explicitly writing the `if` and `else` cases is more readable. But it is just coding-style.